### PR TITLE
Breaking changes

### PR DESCRIPTION
**Pull request by MariekeGO**

### CODE REVIEW

File: release-notes/Breaking_changes/Breaking_changes.md [hunk 1]
```diff
@@ -8,14 +8,27 @@ The following breaking changes have been introduced in recent DataMiner releases
 
 | Release note ID | Release version(s) | Description |
 |--|--|--|
+| [34895](xref:General_Feature_Release_10.3.1#dashboards-app--low-code-apps---gqi-certain-cell-values-in-a-gqi-query-result-would-incorrect-include-the-object-type-id_34895) | DataMiner 10.2.0 [CU1]/10.3.1 | Previously, cell values of GQI result rows for DomInstanceIds, DomDefinitionIds, ProfileInstanceIds, and ProfileDefinitionIds contained both the display value and the object type. Now the GUID value is used as the cell value. |
```
Comment (SebastiaanSL) [inline release-notes/Breaking_changes/Breaking_changes.md:11]: 'contained both the display value of the GUID and the object type. Now only the display value of the GUID is used.' is a bit more clear I think.

File: release-notes/Breaking_changes/Breaking_changes.md [hunk 2]
```diff
@@ -8,11 +8,11 @@ The following breaking changes have been introduced in recent DataMiner releases
 
 | Release note ID | Release version(s) | Description |
 |--|--|--|
-| [34895](xref:General_Feature_Release_10.3.1#dashboards-app--low-code-apps---gqi-certain-cell-values-in-a-gqi-query-result-would-incorrect-include-the-object-type-id_34895) | DataMiner 10.2.0 [CU1]/10.3.1 | Previously, cell values of GQI result rows for DomInstanceIds, DomDefinitionIds, ProfileInstanceIds, and ProfileDefinitionIds contained both the display value and the object type. Now the GUID value is used as the cell value. |
+| [34895](xref:General_Feature_Release_10.3.1#dashboards-app--low-code-apps---gqi-certain-cell-values-in-a-gqi-query-result-would-incorrect-include-the-object-type-id_34895) | DataMiner 10.2.0 [CU1]/10.3.1 | Previously, cell values of GQI result rows for DomInstanceIds, DomDefinitionIds, ProfileInstanceIds, and ProfileDefinitionIds contained both the display value of the GUID and the object type. Now only the display value of the GUID is used as the cell value. |
```
Comment (SebastiaanSL) [inline release-notes/Breaking_changes/Breaking_changes.md:11]: Does the reference also need to start with '#breaking-change'?

Comment (MariekeGO) [reply release-notes/Breaking_changes/Breaking_changes.md]: This release note still had to be adjusted – I've taken care of this now and updated the cross-reference accordingly.

**Review by SebastiaanSL — APPROVED** [review]

File: release-notes/Breaking_changes/Breaking_changes.md [hunk 3]
```diff
@@ -8,14 +8,27 @@ The following breaking changes have been introduced in recent DataMiner releases
...
 | [29095](xref:SRM_1.2.11#breaking-change-quarantining-a-contributing-booking-now-affects-the-main-booking-id_29095) | SRM 1.2.11 | Assigning a quarantined contributing resource to a booking is no longer possible. |
 | [28912](xref:SRM_1.2.11#breaking-change-bookings-that-failed-to-start-now-set-to-failed-id_28912) | SRM 1.2.11 | Custom SRM scripts need to be updated in accordance with SRM_BookingStartFailureTemplate script. |
```
Comment (slc-peter-vanpoucke) [inline release-notes/Breaking_changes/Breaking_changes.md:34]: Rephrase
> Custom SRM scripts need

to

> The custom *Booking Start Failure* script, if present, needs

**Review by slc-peter-vanpoucke — COMMENTED** [review]
I've checked the SRM lines. Looks good, but could you review this one?

File: release-notes/Breaking_changes/Breaking_changes.md (hunk 3 above)
Comment (MariekeGO) [reply release-notes/Breaking_changes/Breaking_changes.md]: Done! Thanks for the feedback!

**Pull request merged**